### ISSUE TITLE
Border is getting disappeared for ".NET Framework/ .NET Core" dropdown and "Create workbook" button upon navigating to them in high contrast black mode.

### ISSUE THREAD

**Issue by WiproPSAccTest** [Microsoft/workbooks#336]
### Steps to Reproduce

Pre-Requisite: Change the system high contrast mode to black before launching the application.
1. Launch Xamarin Workbooks application. Which invokes "Create New Workbook" window.
2. Navigate to ".NET Framwork/.Net core" drop down and "Create workbook" button.
3. Observe the focus around the buttons.

### Expected Behavior
Border should not disappear.
### Actual Behavior
Border is getting disappered for ".NET Framework/ .NET Core" dropdown and "Create workbook" button upon navigating to them in high contrast black mode.

### User Impact
User will not whether it has got the focus or not. So low vision user will be confused.

### Screenshots
![3 border disappeared hcb](https://user-images.githubusercontent.com/22956130/40721912-7f85ad08-6438-11e8-9c85-3c4f5f2e3dfb.png)
![3 normal mode](https://user-images.githubusercontent.com/22956130/40721913-7fb124a6-6438-11e8-81d9-3ec5f243f516.png)
![3 with keyboard focus hcw](https://user-images.githubusercontent.com/22956130/40721915-7fdcc822-6438-11e8-8040-96ebf3141c6e.png)

### Environment
Windows :Windows 10 Pro Version 1803 (OS Build : 17672.1000)
Xamarin Version : 1.4 (master/b3f92f9)

**Comment by WiproPSAccTest** [comment]
Please add the following labels to the bug:
A11yMAS
Accessibility
Xamarin-Workbooks-Windows
Wipro
MAS4.3.1

**Comment by AccessibilityTestingTeam-TCS** [comment]
Exemption Approved - 1813
